docs/_docs/01-ug-quick-start-guide.md

```diff
@@ -59,7 +59,16 @@ Start the Spark Scala shell as follows:
 ```
 
 #### PySpark
-Support for Pyspark is [on the way](https://github.com/microsoft/hyperspace/pull/36).
+Install or upgrade Pyspark by running the following:
+
+```
+pip install --upgrade pyspark
```

Comment:
would it always install pyspark 2.4?

Reply:
Added version 2.4.2 since in sbt script we are using 2.4.2.